### PR TITLE
Added basic routes

### PR DESCRIPTION
**Pull request by lwypchal**
Need to verify cause its with errors

### CODE REVIEW

File: server/dist/container.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"use strict";
```
Comment (fczabator) [inline server/dist/container.js]: All the /dist files shouldn't be pushed to the git repo. Just add them to `.gitignore`.

File: server/src/persistance/schemas/LeagueSchema.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export default new Schema({
+    name: {
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    },
+    discipline: {
```
[the remark this answers is not in the document]
Comment (lwypchal) [reply server/src/persistance/schemas/LeagueSchema.ts]: Yea, it was a custom string in previous version, why not this time?

File: server/src/persistance/schemas/LeagueSchema.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        type: String,
+        required: true,
+    },
+    type: {
```
[the remark this answers is not in the document]
Comment (lwypchal) [reply server/src/persistance/schemas/LeagueSchema.ts]: It is opened to discuss in a group.... for now let's leave this as it is and next we will think in the rankr meeting ;)

File: server/src/persistance/schemas/LeagueSchema.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (fczabator) [reply server/src/persistance/schemas/LeagueSchema.ts]: In order to have a set of determined disciplines. For example to filter by them. Now, one use will type 'squash' and the other 'Squash', which will be two totally separate disciplines in our system. 
But we can change it in the future.

File: server/src/persistance/schemas/LeagueSchema.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (fczabator) [reply server/src/persistance/schemas/LeagueSchema.ts]: ok :)

**Pull request merged**